client/js/libs/handlebars/friendlydate.js

```diff
@@ -4,10 +4,10 @@ const moment = require("moment");
 
 module.exports = function(time) {
 	// See http://momentjs.com/docs/#/displaying/calendar-time/
-	return moment(new Date(time)).calendar(null, {
+	return moment(time).calendar(null, {
 		sameDay: "[Today]",
 		lastDay: "[Yesterday]",
-		lastWeek: "L", // Locale
-		sameElse: "L"
+		lastWeek: "MMMM DD, YYYY",
+		sameElse: "MMMM DD, YYYY"
```

Comment:
Looking at https://en.wikipedia.org/wiki/Date_format_by_country and others, it looks like `1 May 2017` is the most common, `May 1, 2017` is essentially US-only, but `1 May, 2017` exists virtually nowhere.

Do you mind if we go for the first of the 3 versions?